configure

```diff
@@ -173,6 +177,13 @@ while [ $# -ne 0 ]; do
     --with-doxygen=*)
       append_cache_entry Doxygen_ROOT_DIR PATH "$optarg"
       ;;
+    --dev-mode)
+      append_cache_entry CMAKE_BUILD_TYPE STRING Debug
+      append_cache_entry VAST_LOG_LEVEL INTEGER 4
+      append_cache_entry ENABLE_ADDRESS_SANITIZER BOOL true
+      # TODO: eventually, we also want to enable --more-warnings
+      # append_cache_entry MORE_WARNINGS BOOL yes
```

Comment:
Do we have a separate story to make sure we won't forget this?

Reply:
Now we do.